content/en/docs/project-user-guide/application-workloads/deployments.md

```diff
@@ -13,13 +13,14 @@ For more information, see the [official documentation of Kubernetes](https://kub
 ## Prerequisites
 
 - You need to create a workspace, a project and an account (`project-regular`). Please refer to [Create Workspace, Project, Account and Role](../../../quick-start/create-workspace-and-project) if they are not ready yet.
-- You need to sign in with the `project-admin` account and invite `project-regular` to the corresponding project. Please refer to [these steps to invite a member](../../../quick-start/create-workspace-and-project#task-3-create-a-project).
+
+- You need to sign in with the `project-admin` account and invite `project-regular` to the corresponding project. Please refer to [the steps to invite a member](../../../quick-start/create-workspace-and-project#task-3-create-a-project).
 
 ## Create a Deployment
 
 ### Step 1: Open Dashboard
 
-Log in the console as `project-regular`. Go to **Workloads** of a project, choose **Deployments** and click **Create**.
+Log in the console as `project-regular`. Go to **Application Workloads** of a project, choose **Deployments** and click **Create**.
```

Comment:
Same here. Please change it to:

Go to **Application Workloads** of a project, select **Workloads**, and click **Create** under the tab **Deployments**.